### PR TITLE
Fix subprocess wildcard

### PR DESCRIPTION
**Pull request by Mropat**
## Description
*This PR adds/fixes ...*

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

File: cg/meta/rsync.py [hunk 1]
```diff
@@ -48,7 +49,9 @@ def run_rsync_command(self, ticket_id: int = None, dry_run: bool = False) -> Non
 
         if cases[0].data_analysis == Pipeline.SARS_COV_2:
             LOG.info("Delivering report for SARS-COV-2 analysis")
-            covid_report_path = self.covid_report_path % (str(cases[0].internal_id), ticket_id)
+            covid_report_path = glob.glob(
```
Comment (moonso) [inline cg/meta/rsync.py:52]: Would be nice with type hint here

Comment (moonso) [reply cg/meta/rsync.py:52]: suggest to use pathlib instead, example: https://realpython.com/python-pathlib/#counting-files

**Comment by Mropat** [comment]
```
[0|0|955] 3d [maryia.ropart@hasta:~] [S_main] 2m30s $ cg deliver rsync --dry-run 137683
Running CG deliver
Running command rsync -rvL /home/proj/stage/customers/cust124/inbox/137683/ caesar.scilifelab.se:/home/cust124/inbox/137683/
Dry run: process call will not be executed!!
Delivering report for SARS-COV-2 analysis
No report file could be found with path /home/proj/stage/mutant/cases/toplion/results/sars-cov-2_137683_results_*.csv!
[0|0|955] 3d [maryia.ropart@hasta:~] [S_main] 22s $ cd /home/proj/stage/mutant/cases/toplion/results/
-bash: cd: /home/proj/stage/mutant/cases/toplion/results/: No such file or directory
[0|0|952] 3d [maryia.ropart@hasta:~] [S_main] 1 $ mkdir -p /home/proj/stage/mutant/cases/toplion/results/
[0|0|949] 3d [maryia.ropart@hasta:~] [S_main] $ echo hello >> /home/proj/stage/mutant/cases/toplion/results/sars-cov-2_137683_results_122345.csv
[0|0|946] 3d [maryia.ropart@hasta:~] [S_main] $ cg deliver rsync --dry-run 137683
Running CG deliver,
Running command rsync -rvL /home/proj/stage/customers/cust124/inbox/137683/ caesar.scilifelab.se:/home/cust124/inbox/137683/
Dry run: process call will not be executed!!
Delivering report for SARS-COV-2 analysis
Running command rsync -rvL /home/proj/stage/mutant/cases/toplion/results/sars-cov-2_137683_results_122345.csv caesar.scilifelab.se:/home/cust124/inbox/wwLab_automatisk_hamtning/
Dry run: process call will not be executed!!
```

File: cg/meta/rsync.py [hunk 2]
```diff
@@ -38,17 +38,26 @@ def run_rsync_command(self, ticket_id: int = None, dry_run: bool = False) -> Non
...
-        parameters = [rsync_options, delivery_source_path, rsync_destination_path]
+        parameters: List[str] = [rsync_options, delivery_source_path, rsync_destination_path]
         self.process.run_command(parameters=parameters, dry_run=dry_run)
 
         if cases[0].data_analysis == Pipeline.SARS_COV_2:
             LOG.info("Delivering report for SARS-COV-2 analysis")
-            covid_report_path = self.covid_report_path % (str(cases[0].internal_id), ticket_id)
-            covid_destination_path = self.covid_destination_path % customer_id
-            parameters = [rsync_options, covid_report_path, covid_destination_path]
+            covid_report_options: List[str] = glob.glob(
```
Comment (moonso) [inline cg/meta/rsync.py:51]: Ok I suggested to let `self.covid_report_path` be a `Path`-object and then use the `pathlib.Path.glob()` functionality. Feel free to do as you like. I think it is more nice to work with pathlib paths whenever possible

**Review by moonso — APPROVED** [review]
One comment

**Comment by Mropat** [comment]
```
Building wheels for collected packages: cg
  Building wheel for cg (PEP 517) ... done
  Created wheel for cg: filename=cg-20.21.1-py3-none-any.whl size=433437 sha256=ebd6ed108956a9ec63d1dd2735422cdf61b335ed27f97b1af44cdcd0d5fe207b
  Stored in directory: /home/maryia.ropart/tmp/pip-ephem-wheel-cache-d_q6j1nf/wheels/8c/1e/b8/816ed4facf8c5628b55966d3f2448eeb6ca70f35deff59389e
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 20.21.0
    Uninstalling cg-20.21.0:
      Successfully uninstalled cg-20.21.0
Successfully installed cg-20.21.1

```

**Pull request merged**